### ISSUE TITLE
nice grabber lmfao

### ISSUE THREAD

**Issue by DortDev** [shamansid/token-gen-discord#1]
Nice, you put a grabber on git and called it a gen, you're a special one, on the hockey helmet side atleast.

**Comment by Name277** [comment]
> 
> 
> Nice, you put a grabber on git and called it a gen, you're a special one, on the hockey helmet side atleast.

Yep, this skid thinks he is going to grab someone's token from a script that has been published everywhere lol.
